db/data.py

```diff
@@ -418,11 +418,29 @@ def count_lowlevel(mbid):
     """Count number of stored low-level submissions for a specified MBID."""
     with db.engine.connect() as connection:
         result = connection.execute(
-            "SELECT count(*) FROM lowlevel WHERE gid = %s",
+            """SELECT COUNT(*)
+                 FROM lowlevel
+                WHERE gid = %s""",
             (str(mbid),)
         )
         return result.fetchone()[0]
 
+
+def count_many_lowlevel(mbids):
+    """Count number of stored low-level submissions for a specified set
+    of MBID."""
+    with db.engine.connect() as connection:
+        query = text(
+            """SELECT gid
+                    , COUNT(*)
+                 FROM lowlevel
+                WHERE gid
+                   IN :mbids
+             GROUP BY gid;""")
+        return {str(mbid): int(count) for mbid, count
+                in connection.execute(query, mbids=tuple(mbids))}
```

Comment:
I missed this in the first review - we use a dict as the second argument to execute: `{"mbids": tuple(mbids)}`. IN fact, I didn't know that this other syntax was valid, but we should stay consistent.